README.md

```diff
@@ -52,13 +52,13 @@ Complete description of `dagster_polars` behavior for all supported type annotat
 | Type annotation | Behavior                                                                                                                                                                                                              |
 | --- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `DataFrame` | read/write DataFrame. Raise error if it's not found in storage.                                                                                                                                                       |
-| `LazyFrame` | read LazyFrame. Raise error if it's not found in storage.                                                                                                                                                             |
+| `LazyFrame` | read/write LazyFrame. Raise error if it's not found in storage.                                                                                                                                                             |
```

Comment:
Just fyi don't bother updating the README a lot, it's going to be removed in favor of Sphinx docs once `dagster-polars` is merged into Dagster